src/app/pages/WelcomePage/WelcomePage.tsx

```diff
@@ -22,7 +22,7 @@ function WelcomePage(): JSX.Element {
           maxLength={12}
           onChange={(event) => setName(event.target.value)}
         ></input>
-        <Button className={''} type={'submit'} value={'enter'} />
+        <Button type={'submit'} name={'enter'} />
```

Comment:
Doesn’t it work without {} as well? 
```suggestion
        <Button type='submit' name='enter' />
```

Reply:
I'm afraid not - because it's a property, I need to use {}.